cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -287,7 +297,7 @@ mkTracers traceOptions tracer = do
     teeTraceChainTip tverb elided tr = Tracer $ \ev -> do
         traceWith (teeTraceChainTip' tr) ev
         traceWith (teeTraceChainTipElide StructuredLogging tverb elided tr) ev
-        traceWith (teeTraceChainTipElide TextualRepresentation tverb elided (appendName "text" tr)) ev
+        --traceWith (teeTraceChainTipElide TextualRepresentation tverb elided (appendName "text" tr)) ev
```

Comment:
Comment should be removed

Reply:
> Any reason for "(%) = 15.1" instead of simply "15.1%" ?

it is just a "Double" value that is traced and rendered in the logs.

Reply:
> Comment should be removed

this duplicates the traced values in a different context name with textual representation. I am not sure if someone depends on this..